meson.build

```diff
@@ -406,10 +406,12 @@ if build_standalone and get_option('plugin_dell')
 endif
 
 if build_standalone and get_option('plugin_modem_manager')
-  libmm_glib = dependency('mm-glib', version : '>= 1.10.0')
-  add_project_arguments('-DMM_REQUIRED_VERSION="1.10.0"', language : 'c')
+  libmm_glib = dependency('mm-glib', version : '>= 1.17.0')
+  add_project_arguments('-DMM_REQUIRED_VERSION="1.17.1"', language : 'c')
   libqmi_glib = dependency('qmi-glib', version : '>= 1.22.0')
   add_project_arguments('-DQMI_REQUIRED_VERSION="1.23.1"', language : 'c')
+  libmbim_glib = dependency('mbim-glib', version : '>= 1.25.0')
+  add_project_arguments('-DMBIM_REQUIRED_VERSION="1.25.3"', language : 'c')
```

Comment:
Please don't explicitly bump these up unless it's really needed.  Per previous discussion, use a macro to determine version and only run new code when on new version.

Reply:
Do you mean that both libmm_glib and libmbim_glib need macros to control the new code? 
Use #if MM_CHECK_VERSION (1,17,1) ... #endif and #if MBIM_CHECK_VERSION (1,25,3) ... #endif ?

Reply:
The goal is to make sure that this keeps building on all the distros CI has.  So yes; whichever macros are needed to do this